app/src/main/java/org/wikipedia/readinglist/LongPressMenu.kt

```diff
@@ -85,10 +85,15 @@ class LongPressMenu(private val anchorView: View, private val existsInAnyList: B
 
     private fun deleteOrShowDialog() {
         listsContainingPage?.let { list ->
-            RemoveFromReadingListsDialog(list).deleteOrShowDialog(anchorView.context) { _: List<ReadingList?>?, _: ReadingListPage? ->
-                entry?.let {
+            RemoveFromReadingListsDialog(list).deleteOrShowDialog(anchorView.context) { readingLists: List<ReadingList?>?, _: ReadingListPage? ->
```

Comment:
The parameters can actually be non-null.
`readingLists: List<ReadingList?>?, _: ReadingListPage?`
to
`readingLists: List<ReadingList>, _: ReadingListPage`